composer.json

```diff
@@ -34,6 +34,9 @@
         "behatch/contexts": "^2.6",
         "doctrine/doctrine-bundle": "^1.6.3",
         "doctrine/orm": "^2.5.2",
+        "doctrine/mongodb-odm": "^1.1.5",
+        "doctrine/mongodb-odm-bundle": "~3.0",
+        "ext-mongo": "*",
```

Comment:
https://travis-ci.org/api-platform/core/jobs/260125998

> doctrine/mongodb 1.5.0 requires ext-mongo ^1.5 -> the requested PHP extension mongo is missing from your system.

It seem we need both `ext-mongodb` and `ext-mongo` for some reason